templates/kickstart.j2

```diff
@@ -44,8 +58,25 @@ ignoredisk --only-use={{ hostvars[groups[item][0]]['os_disks'] }}
 {% if dhcp_kickstart_install_chrony == false %}
 -chrony
 {% endif %}
+{% if hostvars[groups[item][0]]['kickstart_packages'] is defined %}
+{{ hostvars[groups[item][0]]['kickstart_packages'] }}
+{% endif %}
+{% else %}
+{% if hostvars[groups[item][0]]['kickstart_packages'] is defined %}
```

Comment:
Am I reading something wrong or is the same code there twice?

Reply:
Your eyes are not deceiving you, it's there, but it's only once so it doesn't show up twice after templating is done. 

If dhcp_kickstart_manage_packages is not defined it will throw in some default packages and kickstart_packages.
If dhcp_kickstart_manage_packages is defined it will not throw in some default packages and only use the kickstart_packages.

https://github.com/CSCfi/ansible-role-dhcp-kickstart/blob/468307d1554291d99a83f83fa4468cd02bebb5bc/templates/kickstart.j2#L46

https://github.com/CSCfi/ansible-role-dhcp-kickstart/blob/468307d1554291d99a83f83fa4468cd02bebb5bc/tests/group_vars/compute3/vars.yml#L7-L12

Reply:
Thanks for good review time on this!
I have now updated this bit to have fewer conditionals.
I tried to add some whitespace handling but it messed things up (and the nice ksvalidator complained that the kickstarts were invalid :)